Reviewer/Scripts/controllers.js

```diff
@@ -230,7 +230,6 @@ angular.module('app.controllers', [])
         $scope.initalLoad = true;
         TasteService.GetRoom($stateParams.roomCode, localStorage.getItem("pin"), localStorage.getItem("userId")).then(function (data) {
             $scope.initalLoad = false;
-            data.isBlind = false;
```

Comment:
@Gumbosveins For some reason this was manually set to false, any idea why?

Reply:
No Idea, never got the blind tasting to work properly.